src/labels/CustomLabels.labels

```diff
@@ -1453,7 +1453,7 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Select a desired picklist value for the batch job to selct a value from</shortDescription>
-        <value>Select a field for Preferred Phone to copy the value from when the standard Phone field has no value, or more than one 'Phone' field is defined.</value>
```

Comment:
@robertbwatson can we assume that our translator team's script returned us the single-quoted format here? If so, yipes.

Reply:
Yeah, I saw a lot of instances of this throughout all of the files. A lot of the changes here are just changing single quotes.

Reply:
Yea, I posted this information to the "What Happened" document in quip so it can be addressed.